pkgs/by-name/co/conjure-tor/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+}:
+buildGoModule {
+  name = "conjure-tor";
+  version = "0-unstable-2024-12-22";
```

```suggestion
  pname = "conjure-tor";
  version = "0-unstable-2024-11-11";
```

The date in unstable releases should be the date of commit, not today